src/sign.rs

```diff
@@ -114,17 +112,17 @@ pub mod ed25519 {
     }
 
     #[async_trait]
-    impl Signer for sodiumoxide::crypto::sign::ed25519::SecretKey {
+    impl Signer for ed25519_zebra::SigningKey {
```

Comment:
Can we move this to the tests? Or do we need it here? It's duplicated in librad..